en.txt

```diff
@@ -0,0 +1,78 @@
+Adomas Daškevičius, Vilnius.
+
+About me:
```

Comment:
Its important to mention about your soft skills. For example: you like be a part of team, you are open for feedback, you like collaboration.
Also, you can mention the experience working in startup, collaborating with other students, even the experience collaborating with the mentor.